mod/file/classes/FilePluginFile.php

```diff
@@ -1,42 +1,15 @@
 <?php
 
 /**
- * Override the ElggFile
- *
- * @note When extending an ElggEntity, one should always register a unique type/subtype
- *       combination. We failed to do so here, so get_entity() and friends will always return
- *       these objects as ElggFile. We leave it this way just for BC.
+ * Removed subclass of ElggFile. See https://github.com/Elgg/Elgg/issues/7763
```

Comment:
Add a mention that this was removed in 2.0 so we'll remember to remove it for 3.0.